From 32caa8eafa03dac3252fca05f6b39cc55f9e85a7 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Petr=20P=C3=ADsa=C5=99?= <ppisar@redhat.com>
Date: Mon, 7 Feb 2011 11:37:09 +0100
Subject: [PATCH] Fix spelling in pcretest(1) manual

I kept some non-dictionary words to be consistent.
Credits to John Bradshaw.

Spelling refused by upstream <http://bugs.exim.org/show_bug.cgi?id=1081#c3>:

I have applied these patches, except for "termi-nated", because my
Collins dictionary of spelling and word division shows "termin-ate" and
"termin-ating". I do know that there is a cultural difference between
British and American hyphenation conventions; I try to follow the
British ones, as exemplified in the Collins.
---
 doc/pcretest.1 |    8 ++++----
 1 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/doc/pcretest.1 b/doc/pcretest.1
index beb9d17..8f0552a 100644
--- a/doc/pcretest.1
+++ b/doc/pcretest.1
@@ -467,8 +467,8 @@ recognized:
                after a successful match (number less than 32)
 .\" JOIN
   \eCname     call pcre[16]_copy_named_substring() for substring
-               "name" after a successful match (name termin-
-               ated by next non alphanumeric character)
+               "name" after a successful match (name terminated
+               by next non-alphanumeric character)
 .\" JOIN
   \eC+        show the current captured substrings at callout
                time
@@ -489,8 +489,8 @@ recognized:
                after a successful match (number less than 32)
 .\" JOIN
   \eGname     call pcre[16]_get_named_substring() for substring
-               "name" after a successful match (name termin-
-               ated by next non-alphanumeric character)
+               "name" after a successful match (name terminated
+               by next non-alphanumeric character)
 .\" JOIN
   \eJdd       set up a JIT stack of dd kilobytes maximum (any
                number of digits)
-- 
1.7.7.6

